aspnetcore/security/authentication/index.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Overview of ASP.NET Core Authentication
+author: mjrousos
+description: Learn about authentication in ASP.NET Core.
+ms.author: riande
+ms.custom: mvc
+ms.date: 11/22/2019
+uid: security/authentication/index
+---
+# Overview of ASP.NET Core authentication
+
+By [Mike Rousos](https://github.com/mjrousos)
+
+Authentication is the process of determining a user's identity. [Authorization](xref:security/authorization/introduction) is the process of determining whether a user has access to a resource. In ASP.NET Core, authentication is handled by authentication [middleware](xref:fundamentals/middleware/index). The authentication middleware uses registered authentication handlers to complete authentication-related actions. Examples of authentication-related actions include:
+
```

Comment:
I'm not sure how important the semantics are here.  But technically, authentication is handled by the `IAuthenticationService` which is what uses registered handlers.  The middleware is responsible for 2 things: if a default authenticate scheme is set, it is responsible for setting HttpContext.User.  Also its responsible for routing challenge/requests which are specific for remote authentication flows.  Its probably worth being accurate and spelling out this distinction here.

Reply:
So while most of our implementations of handlers do require our authentication middleware to be present for a mix of not always overlapping reasons, its not actually a requirement at the lowest authentication abstraction layers (IAuthenticationHandler/Schemes).

Reply:
> So while most of our implementations of handlers do require our authentication middleware to be present for a mix of not always overlapping reasons, its not actually a requirement at the lowest authentication abstraction layers (IAuthenticationHandler/Schemes).

That distinction is not meaningful to most users. Telling them to always add the middleware is simpler.

Reply:
I can be more specific about how the handlers are used: 
"In ASP.NET Core, authentication is handled by the `IAuthenticationService` (which is called by authentication middleware). The authentication service uses registered handlers..."

I agree with @Tratcher, though, that for this doc is probably best to just leave it at middleware calls service which calls handlers. As long as developers know those different parts exist, they can learn more about them (and how to use them separate from one another) later.